src/app/shell/admin-tools/data/users/users.component.ts

```diff
@@ -124,6 +124,26 @@ export class UsersComponent implements OnInit, OnDestroy {
     this.onPageChange(PaginationConstants.firstPage);
   }
 
+  /**
+   * This method delete child By Id
+   */
+  public onDeleteChild(child: Child): void {
+    this.matDialog
+      .open(ConfirmationModalWindowComponent, {
+        width: Constants.MODAL_SMALL,
+        data: {
+          type: ModalConfirmationType.deleteChild,
+          property: `${child.firstName} ${child.lastName}`
+        }
+      })
+      .afterClosed()
+      .subscribe((result: boolean) => {
```

Comment:
Since we only perform some action on `true`, we can add `pipe(filter(Boolean))` instead of using `if`